prometheus/src/cluster_metrics.rs

```diff
@@ -0,0 +1,57 @@
+use solana_gossip::cluster_info::ClusterInfo;
+use solana_runtime::bank::Bank;
+
+use crate::{
+    token::Lamports,
+    utils::{write_metric, Metric, MetricFamily},
+};
+use std::{io, sync::Arc, time::SystemTime};
+
+pub fn write_cluster_metrics<W: io::Write>(
```

Comment:
I find all of the “cluster” in this file and especially the metric names a bit confusing. I see Solana calls it `ClusterInfo`, but everything we expose here doesn’t describe the _cluster_, it describes _our node_.

From cluster metrics I would expect things like cluster-wide skip rate, or stake per version. The version of our node is not a property of the cluster, right?